.github/workflows/python-test.yml

```
@@ -142,42 +142,53 @@
+          # Reduce costs by testing one version of Python on MacOS
+          - os: "macos-13"
+            python-version: "3.10"
+          - os: "macos-13"
+            python-version: "3.11"
+    runs-on: ${{ matrix.os }}
+    if: "!startsWith(github.event.head_commit.message, 'Version updated')"
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup
+      - name: Install dependencies
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1090:warning:1:3: ShellCheck can't follow non-constant source. Use a directive to specify location

(shellcheck)